chapters/ch39.md

```diff
@@ -4,31 +4,31 @@
+Giả sử bạn có 200.000 hình ảnh từ internet và 5.000 hình ảnh từ người dùng ứng dụng di động của bạn. Tỷ lệ kích thước giữa các bộ dữ liệu này là 40:1. Về lý thuyết, miễn là bạn xây dựng một mạng neural khổng lồ và huấn luyện nó đủ lâu trên tất cả 205.000 hình ảnh thì sẽ không có vấn đề gì khi cố gắng làm cho thuật toán hoạt động tốt trên cả hình ảnh từ internet và hình ảnh từ di động.
 
 > But in practice, having 40x as many internet images as mobile app images might mean you need to spend 40x (or more) as much computational resources to model both, compared to if you trained on only the 5,000 images.
 
-Nhưng trên thực tế, việc có hình ảnh từ internet gấp 40 lần so với hình ảnh từ ứng dụng di động có thể nghĩa là bạn cần phải sử dụng 40 lần (hoặc nhiều hơn) tài nguyên tính toán để mô hình hóa cả hai, so với nếu bạn chỉ đào tạo trên 5,000 hình ảnh.
+Nhưng trên thực tế, việc có hình ảnh từ internet gấp 40 lần so với hình ảnh từ ứng dụng di động có thể nghĩa là bạn cần phải sử dụng 40 lần (hoặc nhiều hơn) tài nguyên tính toán để mô hình hóa cả hai, so với nếu bạn chỉ đào tạo trên 5.000 hình ảnh.
 
 > If you don’t have huge computational resources, you could give the internet images a much lower weight as a compromise.
 
 Nếu bạn không có tài nguyên tính toán khổng lồ, bạn có thể gán trọng số thấp hơn nhiều cho các hình ảnh từ internet như một cách thỏa hiệp.
 
 > For example, suppose your optimization objective is squared error (This is not a good choice for a classification task, but it will simplify our explanation.) Thus, our learning algorithm tries to optimize:
 
-Ví dụ, giả sử mục tiêu tối ưu của bạn là sai số toàn phương (Đây không phải là một lựa chọn tốt cho một tác vụ phân loại, nhưng nó sẽ đơn giản hóa lời giải thích của chúng ta.) Vì vậy, thuật toán học tập của chúng ta cố gắng tối ưu hóa:
+Ví dụ, giả sử mục tiêu tối ưu của bạn là sai số bình phương (Đây không phải là một lựa chọn tốt cho một tác vụ phân loại, nhưng nó sẽ đơn giản hóa lời giải thích của chúng ta.) Vì vậy, thuật toán học tập của chúng ta cố gắng tối ưu hóa:
 
 ![img](../imgs/C39_01.png)
 
 > The first sum above is over the 5,000 mobile images, and the second sum is over the 200,000 internet images. You can instead optimize with an additional parameter ​𝛽​:
 
-Tổng đầu tiên phía trên là hơn 5,000 hình ảnh từ di động và tổng thứ hai là hơn 200,000 hình ảnh từ internet. Thay vào đó, bạn có thể tối ưu hóa với một tham số bổ sung ​𝛽​:
+Tổng đầu tiên phía trên là trên 5.000 hình ảnh từ di động và tổng thứ hai là hơn 200.000 hình ảnh từ internet. Bạn cũng có thể tối ưu với một tham số bổ sung ​𝛽​:
```

Comment:
Từ "over" ở chỗ over the 200,000 internet images cũng có nghĩa tương tự là "trên".

Để cho rõ nghĩa hơn, anh đề xuất đổi lại câu chữ một chút:

"Trong công thức trên đây, tổng thứ nhất được tính trên 5.000 ảnh từ di động và tổng thứ hai được tính trên 200.000 ảnh từ internet."

Reply:
E vừa update lại r ạ